internal/plugincommon/discovery/discoverytypes/types.go

```diff
@@ -0,0 +1,21 @@
+package discoverytypes
+
+import "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+// Outcome isn't needed for this processor.
```

Comment:
wrong comment?

Reply:
This is referring to the type. I'm not including the contracts in the final outcome.